README.md

```diff
@@ -8,62 +8,36 @@ IdentityModel Extensions for .NET provide assemblies that are interesting for we
 
-## Note about 6.28.0
-We delisted 6.28.0 as we found an edge case where a null reference could occur.
-Please use 6.28.1.
+## IdentityModel 7x
 
-## Note about 6.x
-
-If you noticed, we bumped the release from 5.x to 6.x\
-We were maintaining two releases from two different branches.\
-dev - 6.x\
-dev5x - 5.x
-
-Internally at Microsoft we were quickly required to remove all 3rd party libraries as IdentityModel is all about securing resources.\
-Since there were some breaking changes, given the time-line we had to maintain two releases.
-
-Both of these branches were public and moved forward mostly in lock-step.\
-Once we finished our SignedHttpRequest functionality in the 6.x branch, we realized the delta between 5.x and 6.x was too large to maintain in both branches.\
-We decided now was the time to switch to a single release branch.\
-Since internally the versioning was at 6.4.2, we needed to release at 6.5.0.
-
-## There are some small breaking changes
-
-We built and tested ASP.NET core with 6.5.0 without issues.\
-We also upgraded in place existing applications to 6.5.0 without issues.\
-This of course does not mean you will not hit issues, but we took it seriously.
+We are excited to announce the release of [IdentityModel 7x](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/IdentityModel-7x), a major update to our popular .NET auth validation library. This new version introduces several improvements related to serialization and consistency in the API, which will provide a better user experience for developers, as well as full AOT compatibility on .NET, and huge perf improvements compared to 6x.
 
-Any questions or compatibility problems please open issues [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc).
-
-## Thank you for using our product
-
-The IdentityModel Team.
-
-## Usage
-
-IdentityModel Extensions for .NET 5 has now been released. If you are using IdentityModel Extensions with ASP.NET, the following combinations are supported:
+## Note about 6.x
 
-* **IdentityModel Extensions for .NET 5.x**, **ASP.NET Core 1.x**, **ASP.NET Core 2.x**, **Katana 4.x**
-* **IdentityModel Extensions for .NET 4.x**, **ASP.NET 4**, **Katana 3.x**
-All other combinations aren't supported.
+We bumped the release from 6.x to 7.x.
+We are maintaining two releases from two different branches.
+dev - 7.x
+dev6x - 6.x
 
-For more details see [Migration notes here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/Migrating-from-Katana-(OWIN)-3.x-to-4.x)
+dev6x will be maintained until March 2024, at which point, you will need to move to 7x to continue to get the latest and greatest improvements and security updates.
```

Comment:
would like feedback on this.

Reply:
March = now + 6 months?
Maintain = ? only security fixes?

Reply:
Yes...not sure what was done in the past. I don't think we have any specific guidance on this.

Reply:
We originally committed to 1 year.